view/field/mixin.pug

```diff
@@ -2,6 +2,12 @@ mixin field(label, hotkey, opts = {})
   .field( class=opts.options ? 'is-options' : '' )
     kbd.field_hotkey= hotkey
     input.field_input( aria-label=label )&attributes({ type: 'text', ...opts })
+    if (opts.role === 'spinbutton')
+      .controls
+        button.control.increase( tabindex="-1" )
```

Comment:
Modifiers should be in `is_increase` and elements should have a selector like `BLOCK_ELEMENT` (`.field_control`).

Reply:
Done